pages/standard/rnaseq.tsx

```diff
@@ -32,22 +34,27 @@ function Rnaseq(data: RnaseqProps) {
                 <Row>
                     <span
                         dangerouslySetInnerHTML={{
-                            __html: data.data[0].content.rendered,
+                            __html: props.data[0].content.rendered,
                         }}
                     ></span>
                 </Row>
+                <Row>
+                    <DataSchema schemaData={props.schemaData} />
+                </Row>
             </Container>
             <Footer />
         </>
     );
 }
 
 export const getStaticProps: GetStaticProps = async (context) => {
-    let slugs = ['data-standards-rnaseq-blurb'];
-    let overviewURL = `${WORDPRESS_BASE_URL}${JSON.stringify(slugs)}`;
-    let res = await fetch(overviewURL);
-    let data = await res.json();
-    return { props: { data } };
+    const data = await getStaticContent(['data-standards-rnaseq-blurb']);
+    const schemaData = await getDataSchema(
+        ["bts:ScRNA-seqLevel1", "bts:ScRNA-seqLevel2", "bts:ScRNA-seqLevel3", "bts:ScRNA-seqLevel4"],
+        "https://raw.githubusercontent.com/ncihtan/hsim/master/schema/HTAN.jsonld"
```

Comment:
@milen-sage what would be the best source to get the schema JSON from?

Reply:
@inodb the latest json-ld schema is stored here (it was updated last night for instance):
https://raw.githubusercontent.com/ncihtan/schematic/main/data/schema_org_schemas/HTAN.jsonld

That's the same schema that we use for the data-portal json generation, so it's a single source of truth.